source/_integrations/statistics.markdown

```diff
@@ -30,11 +30,11 @@ The `statistics` integration is different to [Long-term Statistics](https://deve
 
 ## Characteristics
 
-The following statistical characteristics are available. Pay close attention to the right configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
+The following statistical characteristics (`state_characteristic`) are available. Pay close attention to the right configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
```

Comment:
```suggestion
The following statistical characteristics (`state_characteristic`) are available. Pay close attention to the correct configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
```
While maybe not perfect, it seems reasonable to me.

Reply:
Agree with this.

Reply:
```suggestion
The following options for the configuration parameter `state_characteristic` are available as statistical characteristics. Pay close attention to the right configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
```

Reply:
I used a mix of both the suggestions (changing 'right' -> 'correct'), but using the same sentence structure as above.